oscmain.c

```diff
@@ -255,7 +255,7 @@ gint main (int argc, char **argv)
 		if (!ctx)
 			connect_dialog(false);
 		// !!!!!!!
```

Comment:
Is creating a default plot still problematic? If not, you should remove the comment with the exclamation marks.